webpack/assets/javascripts/foreman_settings.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery';
+
+$(document).ready(() => {
+  if (window.location.pathname === window.foreman_url('/settings')) {
+    $('.editable').editable({
```

Comment:
This is probably my lack of knowledge, but how does this get the editable since it's a plugin? Do you need to import the plugin or does it rely on some side effect somewhere?

Reply:
interesting... we require it over the assets pipeline
```js
//= require editable/bootstrap-editable
//= require editable/rails
``` 
in `application.js`, and we load `application.js` after webpack files, therefore it works

Reply:
That sounds rather fragile to rely on the classic pipeline in the webpack pipeline. AFAIK the settings are all defined in Foreman models with the actual values being added by plugins. That means the page is probably not modified by plugins. Wouldn't it be better to rewrite it to react rather than to risk breaking it by moving things if there's no actual gain right now?

Reply:
Also, are we sure editable isn't used by any other code? Any plugins using the `x-editable-rails` gem?